### PR TITLE
Fix create comment bug

### PR DESCRIPTION
**Pull request by bytedeveloperr**

### CODE REVIEW

**Comment by Douglasokolaa** [comment]
This PR, contains changes not related to the ticket. 

Please see documentation you're changing response format on ticket that has been tested and marked as completed already

Please review. This is not compliant with what we all agreed to

https://github.com/hngi/comments-microservice/blob/develop/README.md

**Comment by bytedeveloperr** [comment]
Their methods returns server error, I told them but didnt reply and I'm afraid of still being in stage 3

**Comment by Douglasokolaa** [comment]
Which method in particular. So we look at it. You changed the response format on most method please check

File: app/Http/Controllers/CommentsController.php [hunk 1]
```diff
@@ -206,15 +208,15 @@ public function vote(Request $request, $comment_id)
...
                 'response' => 'error',
             ], 400);
         }
 
         if ($save) {
             return response()->json([
                 'data' => [$comment],
-                'message' => 'Comment voted successfully',
+                'data' => 'Comment voted successfully',
                 'response' => 'Ok'
```
Comment (Douglasokolaa) [inline app/Http/Controllers/CommentsController.php:220]: See here you duplicated data key in the array. And vote method is independent of there's shouldn't give any error. @EphraimDev has already tested this method and graded it

File: app/Http/Controllers/CommentsController.php [hunk 2]
```diff
@@ -138,13 +140,13 @@ public function delete(Request $request, $comment_id)
             $comment->delete();
             return response()->json([
                 'data' => ['comment' => $comment_id],
-                'message' => 'Comment deleted successfully',
+                'data' => 'Comment deleted successfully',
                 'response' => 'Ok'
```
Comment (Douglasokolaa) [inline app/Http/Controllers/CommentsController.php:144]: Why are you removing message returned and assigning the message to data key in the array ?

**Comment by ephraim** [comment]
I haven't tested anything!... I even don't know anything about this project!
Why do you tell, that I've testing here something?
Thats absolutely false!

**Comment by Douglasokolaa** [comment]
> I haven't tested anything!... I even don't know anything about this project!
> Why do you tell, that I've testing here something?
> Thats absolutely false!

Apologies that was a mistake, I've edited it .  I was to make reference to a diff user with similar name

**Pull request closed without merging**